estate/models/estate_property.py

```diff
@@ -0,0 +1,45 @@
+from odoo import models, fields
+from dateutil.relativedelta import relativedelta
+
+
+class estateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate model"
+
+    name = fields.Char(required=True)
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        copy=False, default=fields.Datetime.today() + relativedelta(months=3)
```

Comment:
You can also use fields.Date.today() here which is more appropriate as this is not a datetime field. And the Date field has a [small shortcut](https://github.com/odoo/odoo/blob/c640ca90c6905134c07c41e3820ef67aea38000b/odoo/orm/fields_temporal.py#L32-L33) for the add and sub operations which you can use. Also don't forget the trailing comma


```suggestion
        copy=False, 
        default=fields.Date.add(fields.Date.today(), months=3),
```